packages/salesforcedx-vscode-replay-debugger/src/breakpoints/checkpointService.ts

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { Event, EventEmitter, TreeDataProvider, TreeItem } from 'vscode';
+
+export class CheckpointService implements TreeDataProvider<BaseNode> {
```

Comment:
Just a skeleton - nothing can configure tree items in the checkpoints panel yet.